src/LibraryManager.Vsix/Commands/InstallLibraryCommand.cs

```diff
@@ -105,6 +105,8 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
                     windowInteropHelper.Owner = new IntPtr(hwnd);
 
                     dialog.ShowDialog();
+
+                    Telemetry.TrackUserTask("InstallDialogOpened");
```

Comment:
If it is a user task, I think it's better to use "Open-InstallDialog"

Reply:
This is a menu command invoked by the use. Should it be tracked as an operation?